### ISSUE TITLE
Replace mentions and links to this repo and appdev.openshift.io by attributes

### ISSUE THREAD

**Issue by tradej** [openshiftio/appdev-documentation#489]
All references to this repository in the doc set should be standardised and replaced by attributes. Suggestion:

GitHub:
* `{repo-docs-name}` resolving to `appdev-documentation`
* `{link-repo-docs}` pointing to `https://github.com/openshiftio/appdev-documentation/`

Production:
* `{docs-name}` resolving to `appdev.openshift.io`
* `{link-docs}` pointing to `https://appdev.openshift.io`

In this arrangement, the references to the GitHub repo will look as follows in the code:
```
Modify the link:{link-repo-docs}/blob/master/scripts/deploy_launchpad_mission.sh#L16[`deploy_launchpad_mission.sh`] file in the _{repo-docs-name}_ repository.
```

The references to appdev.openshift.io will look as follows:
```
<1> The directory with the sources of all the __{docs-name}__ guides.
```

**Comment by inoxx03** [comment]
:+1:  Grabbing this...

**Comment by inoxx03** [comment]
Resolved in pr: #516
